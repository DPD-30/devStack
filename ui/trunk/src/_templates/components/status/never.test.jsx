import React from 'react'
import { renderToString } from 'react-dom/server'


import TestComponent from './never.jsx'

jest.unmock('./never.jsx')


test('Status Never matches snapshot', () => {
	const comp = renderToString(<TestComponent />)

	expect(comp).toMatchSnapshot()
})
