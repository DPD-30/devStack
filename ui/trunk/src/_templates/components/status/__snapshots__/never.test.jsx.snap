// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status Never matches snapshot 1`] = `
<span class="badge badge-default"
      data-reactroot
>
  <span aria-hidden="true"
        class="fa fa-star"
  >
  </span>
</span>
`;
